knack/arguments.py

```diff
@@ -256,7 +256,7 @@ def _get_preview_arg_message(self):
             # convert argument dest
             target = '--{}'.format(argument_dest.replace('_', '-'))
         elif options_list:
-            target = sorted(options_list, key=len)[0]
+            target = sorted(options_list, key=len)[-1]
```

Comment:
What will happen if the options_list has 2  longest items with the same length?

Reply:
It will use either of them, which is fine in my mind. I just want to make sure it doesn't use the short option.